help/db_meeting-data.md

```diff
@@ -0,0 +1,39 @@
+### Zawartość tabel
+
+##### Dictionary -- nowe słowniki
+
+| column                   | dictionary_id                        |
+|--------------------------|--------------------------------------|
+| MeetingType.category     | ce12aa23-a5db-49f3-987b-d2db3ab24a3b |
+| Meeting.status           | 3865a3c3-0038-4668-9d55-5d05b79d7fcd |
+| MeetingResource.resource | fdb0f818-aa1e-4ed9-97cc-2a3cb1b702df |
+
+##### MeetingType
+
+| id | facility_id | name               | category_dict_id | default_minutes |
```

Comment:
Kategoria spotkania / typ spotkania - tego nie rozumiem jak to się ma do siebie.

W ogóle to uważam że w takim pliku toby parę słów na górze napisanych po ludzku jak człowiek do człowieka nie zaszkodziło.

Reply:
Zapytaj Ani czy to jest potrzebne im w ogóle, te kategorie.